### DISCUSSION TITLE
Schema Visualizer relationships not showing

### DISCUSSION THREAD

**Discussion by amerryma** [supabase/supabase#18525] [Questions]
Is anyone else having issues with the Schema Visualizer not working when using the Supabse CLI to run studio locally? The issues I'm seeing are:

- No minimap
- Broken relationships (no lines between tables)
- All schemas are being shown (funny enough, relationships work for all but public)

I just build a new project using the latest Supabase CLI. Happy to add more context if needed. Thanks!

**Comment by amerryma** [comment]
I built the CLI from source and updated the studio and pg-meta to the latest docker images but still doesn't work. It seems like studio has a proxy to pg-meta and messes with the query parameters, it seems like pg-meta is working properly.

**Comment by ndr3svt** [comment]
hey there @amerryma , did you find a solution for this issue? I just came across this while refactoring a project, in the local version under Schema Visualizer no lines are drawn to connect the tables, however in the live version it does
